config/quickstart.yaml

```
@@ -1,312 +1,329 @@
-#       dsn: "postgres://analytics:${ANALYTICS_PASSWORD}@analytics-db:5432/metrics?sslmode=require"
-#       # When DSN is specified, server reference is ignored
-#
-# Example 4: No database module
-# =============================
-# modules:
-#   static_files:
-#     # No database section - module won't get database access
-#     config:
-#       static_dir: "/var/www/static"
-
+# HyperSpot Server Configuration
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)